### ISSUE TITLE
GPIO.trig callback level not properly masked on dev-esp32

### ISSUE THREAD

**Issue by stevemac00** [nodemcu/nodemcu-firmware#2883]
Make sure you read and understand http://nodemcu.readthedocs.io/en/dev/en/support/.
Use one of the two templates below and **DELETE THE REST**.

8<------------------------ BUG REPORT -----------------------------------------
### Expected behavior
interrupt callback parameter level is 0 or 256 instead of 0 or 1
### Actual behavior
high level is 256
### Test code
Provide a [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve) which will reproduce the problem.
```Lua
function intr(pin,level)
  print(pin,level)
  gpio.trig(21,gpio.INTR_LOW,intr)
end
gpio.config({gpio={21},dir=gpio.IN, pull=gpio.PULL_UP})
gpio.trig(21,gpio.INTR_HIGH,intr)

-->21	0
-->21	256
```
### NodeMCU version
powered by Lua 5.1.4 on ESP-IDF v3.3-beta1-791-g890a341 on ESP-IDF v3.3-beta1-791-g890a341 on SDK IDF

### Hardware
ESP-WROOM-32
8<------------------------ END BUG REPORT -------------------------------------

**Comment by tomsci** [comment]
Should this have been resolved by the merging of #3086 or is there something else I need to do?

**Comment by marcelstoer** [comment]
Thanks @tomsci
